index.js

```diff
@@ -359,7 +361,7 @@ function makeApiClient(baseUrl, fetch, token) {
                     });
 
                     events.forEach(function(event) {
-                        callback(event.name);
+                        callback(event.name, event);
```

Comment:
> after `"success"` or
`"fail"` events, or after the function returned by the call to `trackJob` is	`"fail"` events, or after the function returned by the call to `trackJob` is
called. It will only be emitted once.	called. It will only be emitted once, and will include no payload

should we return `null` here in such case?

Reply:
Hmm. Possibly. We might revisit this later.